exonum/src/proto/schema/exonum/runtime.proto

```diff
@@ -215,9 +215,25 @@ message InstanceMigration {
 message InstanceState {
   // Service instance specification.
   exonum.runtime.InstanceSpec spec = 1;
-  // Service instance activity status.
+  // Service instance activity status. 
+  //
+  // The `NONE` value means that it is initiated the addition of this instance to the blockchain
```

Comment:
I'd write it something like:

"This field can be `NONE` if instance was created, but activation routine for it is not yet completed. If this field
is set to `NONE`, the `pending_status` must have value `ACTIVE`."